kubeflow/common/cert-manager/Makefile

```diff
@@ -0,0 +1,12 @@
+
+.PHONY: apply
+apply:
+	# Common cert-manager
+	rm -rf $(build_dir) && mkdir -p $(build_dir)
+	kustomize build --load-restrictor LoadRestrictionsNone -o $(build_dir) ./
```

Comment:
nit: I recall that we no longer need load restrictor none, is this case special?

Reply:
Good point, I should clean them up. Done.